fault/fault.go

```diff
@@ -89,12 +89,14 @@ var (
 	ErrInvalidPasswordLength                 = invalidError("invalid password length")
 	ErrInvalidPaymentVersion                 = invalidError("invalid Payment version")
 	ErrInvalidPeerResponse                   = invalidError("invalid peer response")
+	ErrInvalidPhraseLength                   = invalidError("invalid recovery phrase length")
```

Comment:
Error symbol CamelCase and words in the string must match exactly and all words lower case.
(looks like the payment error above has a capital in the string so is also wrong)